### PR TITLE
refactor: better structure links to documentation

### PR DESCRIPTION
**Pull request by ciyer**
# Description

Merge the improvement contributed in  #1724 into the main line of development

### CODE REVIEW

File: client/src/help/Help.present.js [hunk 1]
```diff
@@ -35,8 +35,9 @@ import { WhatsNew1_0_0 as WhatsNew } from "./WhatsNew";
 import { StatuspageDisplay, isStatusConfigured } from "../statuspage";
 import { RenkuNavLink } from "../utils/components/RenkuNavLink";
 import { ExternalDocsLink, ExternalIconLink, ExternalLink } from "../utils/components/ExternalLinks";
+import { Docs, Links, RenkuPythonDocs } from "../utils/constants/Docs";
 
-const discourseUrl = "https://renku.discourse.group";
+const discourseUrl = Links.DISCOURSE;
```
Comment (lorenzo-cavazzi) [inline client/src/help/Help.present.js:40]: This is defined here, but later it's used only once and the other times it's `Links.DISCOURSE`. Would it make sense to not define the variable here and just always use `Links.DISCOURSE` for all the occurrences?

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
Very nice cleanup for the external links! Just a tiny suggestion before merging -- see inline

**Review by lorenzo-cavazzi — APPROVED** [review]

File: client/src/notebooks/SessionCheatSheet.js [hunk 2]
```diff
@@ -218,11 +219,11 @@ function LearnMore() {
       <div>
         For a brief explanation of the commands, refer to the {" "}
         {/* eslint-disable-next-line max-len */}
-        <ExternalDocsLink url="https://raw.githubusercontent.com/SwissDataScienceCenter/renku/master/docs/_static/cheatsheet/cheatsheet.pdf"
+        <ExternalDocsLink url="https://renku-python.readthedocs.io/en/latest/_static/cheatsheet/cheatsheet.pdf"
```
Comment (andre-code) [inline client/src/notebooks/SessionCheatSheet.js:222]: Is there any reason not to include this link in the link list?

**Review by andre-code — DISMISSED** [review]
I love this Refactor, thanks for organizing those links. 🙌

File: client/src/notebooks/SessionCheatSheet.js (hunk 2 above)
Comment (ciyer) [reply client/src/notebooks/SessionCheatSheet.js:222]: Good point!

**Comment by ciyer** [comment]
> I love this Refactor, thanks for organizing those links. 🙌

Was not me, it was @noeleont!

**Review by andre-code — APPROVED** [review]

**Pull request merged**